views/institutional/components/proven/audit-card.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Box,
+  Button,
+  Theme,
+  Typography,
+  useTheme,
+} from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { ArrowRightSVG, MoveBitSVG } from '@/components/svg';
+
+const AuditCard: FC = () => {
+  const theme = useTheme() as Theme;
+
+  return (
+    <Box
+      padding="3xl"
+      display="flex"
+      maxWidth="100%"
+      width="44.25rem"
+      borderRadius="2rem"
+      border={`0.063rem solid ${theme.colors.onPrimary}`}
```

Comment:
This card doesn't looks like the UI on figma, I'm not sure if its border or box shadow but is missing something.